api/PositionOptions.json

```diff
@@ -116,13 +63,61 @@
           "deprecated": false
         }
       },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": "47"
+            },
+            "chrome_android": {
+              "version_added": "47"
+            },
+            "edge": {
+              "version_added": null
```

Comment:
The original note says secure context was not required, yet this isn't `false`. Is there more to the story here?

Reply:
ditto